nvim-types/src/string.rs

```diff
@@ -111,6 +110,12 @@ impl Clone for String {
     }
 }
 
+impl Drop for String {
+    fn drop(&mut self) {
+        unsafe { std::ptr::drop_in_place(self.data) }
```

Comment:
again, `drop_in_place` doesn't free the value, it just destructor of `c_char` for first element which is a no op

Reply:
fixed in d86ec1e58179a132afaa28ca68637fa1eadaaffc